%YAML:1.0
#
#    Copyright (c) 2017 Mobile Robots Laboratory at Poznan University of Technology:
#    -Jan Wietrzykowski name.surname [at] put.poznan.pl
#
#    Permission is hereby granted, free of charge, to any person obtaining a copy
#    of this software and associated documentation files (the "Software"), to deal
#    in the Software without restriction, including without limitation the rights
#    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
#    copies of the Software, and to permit persons to whom the Software is
#    furnished to do so, subject to the following conditions:
#
#    The above copyright notice and this permission notice shall be included in all
#    copies or substantial portions of the Software.
#
#    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
#    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
#    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
#    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
#    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
#    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
#    SOFTWARE.

map:
  
  readFromFile: 1

  mapFiles:
#    - "/mnt/data/TERRINet/scenes/scene0000_00/scene0000_00.map"
    - "/mnt/data/TERRINet/scenes/scene0001_00/scene0001_00.map"
#    - "/mnt/data/TERRINet/scenes/scene0002_01/scene0002_01.map"

matching:

  planeToPlaneAngThresh: 0.213
  planeDistThresh: 15.0
  planeToPlaneDistThresh: 0.802

planeSlam:

  # TERRINet
  cameraMatrix: !!opencv-matrix
    rows: 3
    cols: 3
    dt: f
    data: [ 496.635831, 0.0, 334.972011,
            0.0, 496.635831, 248.444878,
            0.0, 0.0, 1.0 ]

  nrows: 480

  ncols: 640
